src/test/java/nextstep/subway/acceptance/LineAcceptanceTest.java

```diff
@@ -140,29 +118,30 @@ void deleteLine() {
 
         // when
         String uri = createResponse.header("Location");
-        ExtractableResponse<Response> response = RestAssured.given().log().all()
-            .when()
-            .delete(uri)
-            .then().log().all()
-            .extract();
+        ExtractableResponse<Response> response = deleteLineRequest(uri);
 
         // then
         assertThat(response.statusCode()).isEqualTo(HttpStatus.NO_CONTENT.value());
     }
 
-    private ExtractableResponse<Response> createLineRequest(String name, String color) {
-        Map<String, String> params = new HashMap<>();
-        params.put("name", name);
-        params.put("color", color);
+    /**
+     * Given 지하철역 생성을 요청 하고
+     * When 같은 이름으로 지하철역 생성을 요청 하면
+     * Then 지하철역 생성이 실패한다.
+     */
+    @DisplayName("중복이름으로 지하철 노선 생성")
```

Comment:
```suggestion
    @DisplayName("중복이름으로 지하철 노선 생성 실패")
```
시나리오가 주석으로 있긴 하지만 displayName으로 어떤 테스트를 하는지 조금 더 상세하게 적어주셔도 좋을 것 같아요 :)